### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by edavidcsoft**

### CODE REVIEW

**Comment by edavidcsoft** [comment]
CLA Agreement signed

File: content/en/docs/tutorials/online-training/overview.md [hunk 1]
```diff
@@ -15,10 +15,14 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Amazon EKS Deep Dive with Hands-On Labs (Linux Academy)] (https://linuxacademy.com/amazon-web-services/training/course/name/amazon-eks-deep-dive)
 
+* [Online Kubernetes Training with CKA & CKAD prep (Component Soft)](https://www.componentsoft.io/kubernetes-administration-training-2/)
```
Comment (sftim) [inline content/en/docs/tutorials/online-training/overview.md]: The existing list is sorted alphabetically. Add new entries sorted into the correct location.

Comment (edavidcsoft) [reply content/en/docs/tutorials/online-training/overview.md]: Sorting fixed

**Comment by sftim** [comment]
Can you rebase this against _master_ @edavidcsoft ? https://kubernetes.io/docs/contribute/intermediate/#work-on-the-local-repository has tips on how to do that.

**Comment by Rajakavitha1** [comment]
/close
@edavidcsoft please rebase this PR as suggested by @sftim .
Closing this PR for now. Please feel free to reopen after the rebase.

**Pull request closed without merging**